features/admin/admin.module.css

```
@@ -0,0 +1,889 @@
+  --admin-supported: #5f8164;
+  --admin-gated: #b0713d;
+  --admin-deferred: #81766a;
+  --admin-legacy: #756b82;
+  --admin-privacy: #84586b;
+  --admin-failed: #a9504d;
+  min-height: 100svh;
+  display: grid;
+  grid-template:
+    "topbar topbar" 57px
+    "sidebar workspace" minmax(calc(100svh - 57px), auto)
+    / 220px minmax(0, 1fr);
```

Stylelint (17.14.0)

[error] Unexpected newline before "/" (scss/operator-no-newline-before)

(scss/operator-no-newline-before)